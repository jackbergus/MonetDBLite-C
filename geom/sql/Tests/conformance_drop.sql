DROP TABLE map_neatlines;
DROP TABLE named_places;
DROP TABLE ponds;
DROP TABLE buildings;
DROP TABLE streams;
DROP TABLE bridges;
DROP TABLE forests;
DROP TABLE divided_routes;
DROP TABLE road_segments;
DROP TABLE lakes;
DROP TABLE geometry_columns;
DROP TABLE spatial_ref_sys;
