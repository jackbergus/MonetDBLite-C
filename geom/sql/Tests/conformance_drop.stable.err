stderr of test 'conformance_drop` in directory 'geom/sql` itself:


# 15:50:10 >  
# 15:50:10 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=39855" "--set" "mapi_usock=/var/tmp/mtest-9666/.s.monetdb.39855" "--set" "monet_prompt=" "--forcemito" "--set" "mal_listing=2" "--dbpath=/ufs/sjoerd/Monet-devel/var/MonetDB/mTests_geom_sql" "--set" "mal_listing=0"
# 15:50:10 >  

# builtin opt 	gdk_dbpath = /ufs/sjoerd/Monet-devel/var/monetdb5/dbfarm/demo
# builtin opt 	gdk_debug = 0
# builtin opt 	gdk_vmtrim = no
# builtin opt 	monet_prompt = >
# builtin opt 	monet_daemon = no
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_open = false
# builtin opt 	mapi_autosense = false
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	mapi_open = true
# cmdline opt 	mapi_port = 39855
# cmdline opt 	mapi_usock = /var/tmp/mtest-9666/.s.monetdb.39855
# cmdline opt 	monet_prompt = 
# cmdline opt 	mal_listing = 2
# cmdline opt 	gdk_dbpath = /ufs/sjoerd/Monet-devel/var/MonetDB/mTests_geom_sql
# cmdline opt 	mal_listing = 0
# cmdline opt 	gdk_debug = 536870922

# 15:50:10 >  
# 15:50:10 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-9666" "--port=39855"
# 15:50:10 >  


# 15:50:10 >  
# 15:50:10 >  "Done."
# 15:50:10 >  

