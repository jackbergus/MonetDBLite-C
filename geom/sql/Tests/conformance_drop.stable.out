stdout of test 'conformance_drop` in directory 'geom/sql` itself:


# 15:50:10 >  
# 15:50:10 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=39855" "--set" "mapi_usock=/var/tmp/mtest-9666/.s.monetdb.39855" "--set" "monet_prompt=" "--forcemito" "--set" "mal_listing=2" "--dbpath=/ufs/sjoerd/Monet-devel/var/MonetDB/mTests_geom_sql" "--set" "mal_listing=0"
# 15:50:10 >  

# MonetDB 5 server v11.18.0 (hg id: 5d7fe15187f5)
# This is an unreleased version
# Serving database 'mTests_geom_sql', using 8 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 64bit OIDs dynamically linked
# Found 15.591 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2014 MonetDB B.V., all rights reserved
# Visit http://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://madrid.ins.cwi.nl:39855/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-9666/.s.monetdb.39855
# MonetDB/GIS module loaded
# MonetDB/JAQL module loaded
# MonetDB/SQL module loaded

Ready.

# 15:50:10 >  
# 15:50:10 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-9666" "--port=39855"
# 15:50:10 >  

#DROP TABLE map_neatlines;
#DROP TABLE named_places;
#DROP TABLE ponds;
#DROP TABLE buildings;
#DROP TABLE streams;
#DROP TABLE bridges;
#DROP TABLE forests;
#DROP TABLE divided_routes;
#DROP TABLE road_segments;
#DROP TABLE lakes;
#DROP TABLE geometry_columns;
#DROP TABLE spatial_ref_sys;

# 15:50:10 >  
# 15:50:10 >  "Done."
# 15:50:10 >  

